docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - `Improvements` - Unuseful log about missed i18n section has been removed  [#1269](https://github.com/codex-team/editor.js/issues/1269)
 - `Fix` - Fixed issue with enter key in inputs and textareas [#920](https://github.com/codex-team/editor.js/issues/920)
 - `Improvements` - Allowed to set `false` as `toolbox` config in order to hide Toolbox button [#1221](https://github.com/codex-team/editor.js/issues/1221)
+- `Fix` - Fixed bug of getBlockByIndex API[#1270](https://github.com/codex-team/editor.js/issues/1270)
```

Comment:
Maybe try writing a more expressive description. Also, put a full stop at the end and a blank before the link.